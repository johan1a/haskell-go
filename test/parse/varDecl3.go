package main;
var abc1 int = 3
var abc2 int = 2
func main() {
fmt.Print (2 - 1)

}
