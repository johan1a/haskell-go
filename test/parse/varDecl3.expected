SourceFile (Package "main") [TopLevelDecl1 (VarDecl [IdDecl "abc1"] (TypeName (TypeNameIdentifier "int")) [UnaryExpr (PrimaryExpr (PrimaryExpr1 (Operand1 (BasicLit (IntLit 3)))))]),TopLevelDecl1 (VarDecl [IdDecl "abc2"] (TypeName (TypeNameIdentifier "int")) [UnaryExpr (PrimaryExpr (PrimaryExpr1 (Operand1 (BasicLit (IntLit 2)))))]),TopLevelDecl2 (FunctionDecl2 "main" (Signature1 []) (Block [SimpleStmt (ExpressionStmt (UnaryExpr (PrimaryExpr (PrimaryExpr7 (PrimaryExpr1 (Operand2 (OperandName2 (QualifiedIdent "fmt" "Print")))) (Arguments5 [BinExpr (AritmExpr (SubExpr (UnaryExpr (PrimaryExpr (PrimaryExpr1 (Operand1 (BasicLit (IntLit 2)))))) (UnaryExpr (PrimaryExpr (PrimaryExpr1 (Operand1 (BasicLit (IntLit 1))))))))])))))]))]
